system/Controller.php

```diff
@@ -179,14 +179,14 @@ protected function loadHelpers()
 	 * A shortcut to performing validation on input data. If validation
 	 * is not successful, a $errors property will be set on this class.
 	 *
-	 * @param array $rules
-	 * @param array $messages An array of custom error messages
+	 * @param array|string  $rules
+	 * @param array         $messages An array of custom error messages
 	 *
 	 * @return boolean
 	 */
 	public function validate($rules, array $messages = []): bool
 	{
-		$this->validator = Services::validation();
+		$this->validator = \Config\Services::validation();
```

Comment:
Would prefer this was added at the top of the as a use statement. Not sure why it wasn't already.